boards/nordic/nrf54l15pdk/Kconfig.nrf54l15pdk

```diff
@@ -5,3 +5,12 @@ config BOARD_NRF54L15PDK
 	select SOC_NRF54L15_ENGA_CPUAPP if BOARD_NRF54L15PDK_NRF54L15_CPUAPP || BOARD_NRF54L15PDK_NRF54L15_CPUAPP_NS
 	select SOC_NRF54L15_ENGA_CPUFLPR if BOARD_NRF54L15PDK_NRF54L15_CPUFLPR || \
 					    BOARD_NRF54L15PDK_NRF54L15_CPUFLPR_XIP
+
+if SOC_NRF54L15_ENGA_CPUFLPR && PARTITION_MANAGER_ENABLED
+config FLASH_BASE_ADDRESS
+	hex
+	default 0x0
+config FLASH_SIZE
+	int
+	default 1524
+endif
```

Comment:
this seems like something which should be in the `Kconfig.defconfig`.
The `Kconfig.nrf54l15pdk` is shared with sysbuild. which doesn't want to know about `FLASH_SIZE` and `FLASH_BASE_ADDRESS`.